Neos.ContentRepositoryRegistry/Configuration/Objects.yaml

```diff
@@ -42,6 +42,11 @@ Neos\ContentRepository\Feature\ContentStreamCommandHandler:
 Neos\ContentRepositoryRegistry\Legacy\ObjectFactories\DbalContentGraphObjectFactory:
   scope: singleton
 
+Neos\ContentGraph\DoctrineDbalAdapter\Domain\Repository\ContentGraph:
```

Comment:
This seems to be the only thing added sofar ...

Reply:
im just wondering why in this PR?

Reply:
basically because otherwise it won't work ^^